.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+version: 2.1
+
+references:
+  environment: &environment
+    IMAGE_NAME: "zappi/fsfreeze"
+
+jobs:
+  test:
+    docker:
+      - image: docker:18.06.3-ce-git
+    environment: *environment
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+          version: 18.06.0-ce
+      - run:
+          name: "Build Image"
+          command: docker build -t $IMAGE_NAME .
+      - run:
+          name: "Test Image"
+          command: docker run $IMAGE_NAME fsfreeze --version
+
+workflows:
+  version: 2
```

Comment:
According to CircleCI workflow number isn't actually a thing in config version `2.1` anymore
<img width="737" alt="Screenshot 2019-06-10 at 18 13 45" src="https://user-images.githubusercontent.com/17402708/59209828-8136d180-8bac-11e9-90d3-1c0fe9b72b4d.png">
https://discuss.circleci.com/t/circleci-2-1-config-overview/26057

Reply:
Addressed in https://github.com/Intellection/docker-fsfreeze/pull/1/commits/5e5df4ad8cd1e367280f308f59b46f3169d36ee6.